script.js

```diff
@@ -1,7 +1,533 @@
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+  },
+
+  {
+    title: 'Data Dashboard Healthcare',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#5',
+    'link to source': '#6',
+  },
+
+  {
+    title: 'Website Portfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#7',
+    'link to source': '#8',
+  },
+
+  {
+    title: 'Keeping track of hundreds of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    tech: ['html', 'Bootstrap', 'Ruby on rails'],
+    'live button': 'See live',
+    'source button': 'See Source',
+    'image desktop': 'images/Snapshoot.png',
+  },
+
+];
+
+const p1 = projects_container[0];
+const p2 = projects_container[1];
+const p3 = projects_container[2];
+const p4 = projects_container[3];
+const p5 = projects_container[4];
+
+// first mobile project
+const div21 = document.getElementById('ret');
+const img6 = document.createElement('img');
+img6.src = p1['image mobile'];
+img6.alt = 'alt';
+div21.appendChild(img6);
+
+const div22 = document.getElementById('works_mobile');
+const h38 = document.createElement('h3');
+h38.className = 'heed';
+h38.innerText = p1.title;
+div22.appendChild(h38);
+const para9 = document.createElement('p');
+para9.innerText = p1.description;
+div22.appendChild(para9);
+
+const div23 = document.getElementById('works2');
+const div24 = document.createElement('div');
+div24.className = 'sites';
+const tech9 = document.createElement('ul');
+for (const i of p1.tech) {
+  const li = document.createElement('li');
+  li.innerText = i;
+  tech9.appendChild(li);
+}
+div24.appendChild(tech9);
+div23.appendChild(div24);
+const button10 = document.createElement('button');
+button10.type = 'button';
+button10.className = 'button1';
+button10.innerText = p1.button;
+button10.onclick = function () { myFunction4(); };
+div23.appendChild(button10);
+
+// rendering mobile projects
+function renderProject(p) {
```

Comment:
- [ ] You page becomes totally unresponsive once I click See project. Give it a fix.

![Screenshot from 2021-09-03 12-43-46](https://user-images.githubusercontent.com/65501535/131986090-184f3c1b-2dac-432d-9d62-26e691256792.png)

Reply:
Okay, once you click on the 'see project' button, the popup menu appears and the scroll is disabled until you click on the "x" icon, then everything goes to normal.

Reply:
Yes, But it scrolls all the way up to the top of the page and like you said, scroll is disabled. I had to zoom out from 100% to 30% to be able to see your popup window.

Reply:
![zoom at 30% png](https://user-images.githubusercontent.com/85113567/132008410-ed04cdeb-7b50-4b03-8ea7-34c7d9f4abea.jpg)
This is my zoom at 30% and the popup remains at the top with no need to scroll before seeing it so any issue might be coming from your end.